src/theme/colors.js

```diff
@@ -64,7 +64,7 @@ const colors = {
   },
   alert: {
     base: "#D93A3A",
-    dark: "#AE393A",
+    dark: "#D93A3A",
```

Comment:
Can you use the base property instead of modifying `dark`?

Reply:
That was a design request I believe

Reply:
I think for this case you should pass a style prop, unfortunately

Reply:
I've reverted this change